integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/ssl/axis2.xml

```diff
@@ -619,8 +615,8 @@
 
             <handler name="HTTPLocationBasedDispatcher"
                      class="org.apache.axis2.dispatchers.HTTPLocationBasedDispatcher"/>
-            <handler name="MultitenantDispatcher"
-                     class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/>
+            <!--handler name="MultitenantDispatcher"
+                     class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/-->
```

Comment:
Shall we remove this instead of commenting out?

Reply:
Done in #575